feature-libs/organization/administration/components/unit/details/unit-details.component.html

```diff
@@ -1,9 +1,11 @@
 <cx-org-card
   *ngIf="model$ | async as model"
   i18nRoot="orgUnit.details"
+  [cxFocus]="{ refreshFocus: model }"
   [showHint]="true"
 >
   <a
+    *ngIf="model.active"
```

Comment:
Is this change necessary? We are using already class=disabled for that unable to open edit form.
I tnink it can be breaking change, and doesnt look good for mobile devices

Reply:
probably not actually. This was apparently added previously to mitigate a behaviour that I fixed in the focus directive. I wasn't aware of the ngif being added.